pkgs/development/python-modules/pytensor/default.nix

```diff
@@ -16,21 +16,22 @@
 , pytestCheckHook
 , pythonOlder
 , scipy
+, tensorflow-probability
 , typing-extensions
 }:
 
 buildPythonPackage rec {
   pname = "pytensor";
-  version = "2.11.3";
+  version = "2.17.0";
   format = "setuptools";
 
-  disabled = pythonOlder "3.7";
+  disabled = pythonOlder "3.9";
 
   src = fetchFromGitHub {
     owner = "pymc-devs";
-    repo = pname;
+    repo = "pytensor";
     rev = "refs/tags/rel-${version}";
-    hash = "sha256-4GDur8S19i8pZkywKHZUelmd2e0jZmC5HzF7o2esDl4=";
+    hash = "";
```

Comment:
Evaluation is broken because the source hash is missing.

Reply:
Yes indeed ! Fixed the hash. However, I still cannot build because tensorflow (and thus tensorflow-probability) is broken.
What would you think about disabling those very few tests temporarily ?

Reply:
It looks like the tests check for tensorflow-probability and skip if it's not installed, so I'd be fine removing `tf*` as dependencies if this actually works, otherwise disabling those tests.

Reply:
Ok fine, let's do that.